src/app/static/src/app/components/reviewInputs/ReviewInputs.vue

```diff
@@ -51,11 +54,18 @@ export default defineComponent({
     },
     setup() {
         const store = useStore<RootState>();
-        const activePlot = ref<InputPlotName>(inputPlotNames[0]);
+        const hasTimeSeries = computed(() => !!store.state.surveyAndProgram.anc || !!store.state.surveyAndProgram.program)
+        const availablePlots = computed(() => {
+            if (!hasTimeSeries.value) {
+                return inputPlotNames.filter(name => name != "timeSeries")
+            } else {
+                return inputPlotNames
+            }
+        })
+        const activePlot = ref<InputPlotName>(availablePlots.value[0]);
```

Comment:
i think we can merge `hasTimeSeries` into `availablePlots` we dont actually need this to be computed, since `activePlot` is fully determined by `availablePlots` we can get rid of the `&& hasTimeSeries` in the template above